glide.yaml

```diff
@@ -13,7 +13,7 @@ import:
   - builder/dockerfile/parser
 - package: github.com/fsouza/go-dockerclient
   repo:    https://github.com/openshift/go-dockerclient.git
-  version: openshift-3.9
+  version: openshift-4.0
```

Comment:
reason for not floating on `master`?

Reply:
in general we never want to float on master if we have a choice.  Even the other ones i switched to master is because they don't have release branches and at some point (closer to shipping) we'll probably want to switch them to specific commits so they don't get updated when someone updates a dep.